var failingCode = require("./")

process.on('uncaughtException', function (error) {
  console.log(failingCode(error))
});

var foo
hereIfail++
var bar
